pkg/apis/nsx.vmware.com/v1alpha1/externaladdressbinding_types.go

```diff
@@ -0,0 +1,39 @@
+package v1alpha1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type ExternalAddressBindingSpec struct {
+	VMName        string `json:"vmName"`
+	InterfaceName string `json:"interfaceName"`
```

Comment:
Could we make it optional, and if not set use the first interface of the VM (given most VMs have only a single interface)?

Reply:
Added `omitempty` for this.